js/repl/ReplOptions.js

```diff
@@ -450,7 +450,7 @@ const styles = {
     }
   }),
   inputCheckboxLeft: css({
-    marginRight: '0.5rem'
+    margin: '0 0.5rem 0 0 !important' // Clashes with default input[type="checkbox"] styles in main.css
```

Comment:
Is it possible to make the selector more specific instead of using `!important`?